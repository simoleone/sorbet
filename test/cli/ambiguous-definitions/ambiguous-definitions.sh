#!/bin/bash

set -e

main/sorbet --silence-dev-message --stripe-mode test/cli/ambiguous-definitions/foo.rb 2>&1

