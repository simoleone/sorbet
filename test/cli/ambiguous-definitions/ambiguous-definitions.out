test/cli/ambiguous-definitions/foo.rb:5: Class definition is ambiguous https://srb.help/5068
     5 |    module Foo::Bar
     6 |    end
    test/cli/ambiguous-definitions/foo.rb:4: Previously defined here.
     4 |  module Foo
          ^^^^^^^^^^
Errors: 1
